docs/source/en/quicktour.mdx

```diff
@@ -14,53 +14,22 @@ specific language governing permissions and limitations under the License.
 
 [[open-in-colab]]
 
-Get up and running with 🤗 Transformers! Start using the [`pipeline`] for rapid inference, and quickly load a pretrained model and tokenizer with an [AutoClass](./model_doc/auto) to solve your text, vision or audio task.
+The 🤗 Transformers library is built around four major concepts to maximize user-friendliness and flexibility:
```

Comment:
This is not completely correct: the Transformers library is not built around those four features. I'd keep the introduction line around something like "Get up and running with 🤗 Transformers" as before.